CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 1.9.0 - 2022-11-16
+
+### Added
+
+- Create Function button in Workspace view [#3430](https://github.com/microsoft/vscode-azurefunctions/pull/3430)
+- Warning when a stack has an upcoming end of life [#3382](https://github.com/microsoft/vscode-azurefunctions/pull/3382)
+- Updated templates to enable dynamic concurrency [#3365](https://github.com/microsoft/vscode-azurefunctions/pull/3365)
+
+### Changed
+
+- Always on = on when creating an Azure function on an app service plan [#3422](https://github.com/microsoft/vscode-azurefunctions/pull/3422)
```

Comment:
```suggestion
- Always on = on when creating an Azure Function App on an app service plan [#3422](https://github.com/microsoft/vscode-azurefunctions/pull/3422)
```

Just for branding reasons, Azure Function App should be capitalized if prefixed with Azure. If it's just function app, then it's okay to lower case.